### ISSUE TITLE
BRD enhancing songs use the skill of whatever is in the range slot, regardless of what it is

### ISSUE THREAD

**Issue by almuth150** [DerpyProjectGroup/topaz#5124]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce

Equip a non-instrument in the range slot (say, COR/BRD with a gun) and do a buff song, then compare the boost you get from it to doing the same buff song with nothing in the range slot.

## Expected behavior

The skill of the item in the range slot should only be used if an instrument is equipped.

**Comment by overefined** [comment]
```
if caster:isPC() then
        -- Add ranged skill level ONLY if it's an instrument.
        local rangeType = caster:getWeaponSkillType(xi.slot.RANGED)

        -- String instruments have half the skill effectiveness and amplify the AoE in exchange.
        if rangeType == xi.skill.WIND_INSTRUMENT then
            singingLvl = singingLvl + caster:getSkillLevel(rangeType)
        elseif rangeType == xi.skill.STRING_INSTRUMENT then
            singingLvl = singingLvl + math.floor(caster:getSkillLevel(rangeType) / 2)
        end
    else
        singingLvl = singingLvl * 2
    end
```

**Comment by almuth150** [comment]
Right, this got fixed some time after I made the issue. https://github.com/LandSandBoat/server/commit/6afc5dcfb12da1d02a24bb55ee596ff61f794104
